source/_components/google_assistant.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: "Google Assistant"
+description: "Setup for Google Assistant integration"
+date: 2017-10-17 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google-assistant.png
+ha_category: Voice
+featured: true
+ha_release: 0.56
+---
+
+# Google Assistant Docs
+The Google Assistant component requires a bit more setup than most due to the way Google requires Assistant Apps to be set up.
+
+### {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+google_assistant:
+  project_id: someproject-2d0b8
+  client_id: [long random string]
+  access_token: [different long random string]
+  exposed_domains:
+    - switch
+    - light
+    - group
+```
+
+*Note:* It's very important that you use something very long strings for `client_id` and `access_token`. Those are essentially credentials to your Home Assistant instance.
+
+*Configuration Variables:*
+* *expose_by_default* (Optional): Expose devices in all supported domains by default.
+* *project_id* (Required): Project ID from the Google Developer console (looks like `words-2ab12`)
+* *client_id* (Required): A long random url safe string (no spaces or special characters) that will be used for Implicit OAuth.
+* *access_token* (Required): Another long random url safe string.
+* *exposed domains* (Optional): An array of Home Assistant domains to expose to Google Assistant.
```

Comment:
Wasn't optional in the first version (Was spewing errors when I didn't have it), not sure about the new version?

Reply:
I'm not using `exposed_domains` and mine's working ok so that may have been an old version issue (I just noticed there's a missing underscore here in the doc though)